Update GPRS_Shield_Arduino.cpp

nscnm87 opened the pull request:
Hi,
I had a problem with the library because, though the SIM is in the original country, it was in roaming due to agreements with others operators in case there is no signal and anyway in Europe there are no costs for roaming so someone may want to use SIM card in another country.

ortegafernando commented:
Updated with no ROAMING option, directly it checks 0,1 or 0,5. 
Do you think that is ok?

nscnm87 commented:
I always need it with roaming active but I thought that someone may want to
deactivate it.

As you prefer.

I have added one more function to my local copy, to check the status of an
outcaming call. It check the answer and return different codes if the
sim900 return BUSY or NO CARRIER.

I'm not very good in C but it's working, I can add it to the library and
someone can correct it

Il 01 gen 2018 20:42, "ortegafernando" <notifications@github.com> ha
scritto:

Updated with no ROAMING option, directly it checks 0,1 or 0,5.
Do you think that is ok?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Seeed-Studio/GPRS_SIM900/pull/23#issuecomment-354672199>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/ADOiu5sKCVUk0VRtr_6Z_s4yWvHzkStvks5tGTUqgaJpZM4RP1o3>
.
